vacasa/connect/connect.py

```diff
@@ -916,8 +914,10 @@ def create_reservation_import(self,
                                   created_by: int = None,
                                   currency_code: str = None,
                                   display_currency_code: str = None,
+                                  email: str = None,
```

Comment:
Just double-checking:  does connect handle null vs missing in the way that you want here?  It looks like we'll be passing a null here on every request (rather than just not providing an email parameter).  Not sure if that matters.  Same question w/ notes

Reply:
Resolved.